c_cxx/lesson2/switch/Homework_IM-19.02/main.cpp

```diff
@@ -0,0 +1,53 @@
+#include <iostream>
+
+enum Months {
+    January,
+    February,
+    March,
+    April,
+    May,
+    June,
+    July,
+    August,
+    September,
+    October,
+    November,
+    December
+};
+
+
+char printMonth(Months month){
```

Comment:
What char do you return, could `void` type be better?
```suggestion
void printMonth(Months month){
```